include/compartment.h

```diff
@@ -29,11 +29,30 @@
 // about
 
 struct func_intercept;
+void compartment_transition_out();
+int64_t comp_exec_in(void*, void* __capability, void*);
+void comp_exec_out();
 
+// Declare built-in function for cache synchronization:
+// https://community.arm.com/arm-community-blogs/b/architectures-and-processors-blog/posts/caches-and-self-modifying-code
+extern void __clear_cacher(char*, char*);
```

Comment:
cacher -> cache

It's spelt correctly where used, though, so I'm not sure what's going on there.

Reply:
That is concerning, that using `clear_cache` didn't flag an error, while the definition was wrong. I have fixed the definition and the type in https://github.com/capablevms/CHERI-ELF-comp/pull/2/commits/d919632f88efccb19080646b089771965c006726 and https://github.com/capablevms/CHERI-ELF-comp/pull/2/commits/ab96f6fb96ea5eb6a67e655f1e7c20fc5e5e1dae.